### PR TITLE
[Enhancement] Run loop when TemporyTarget is changed

### PR DESCRIPTION
**Pull request by harispoljo**
See #1626

### CODE REVIEW

**Comment by MilosKozak** [comment]
@AdrianLxM @jotomo 
I personaly have no problem with it. I'm only not sure if there is some significant impact

**Comment by TebbeUbben** [comment]
I'm okay with it as well.

**Comment by jotomo** [comment]
* With a high target, the loop might set a zero temp faster (ok)
* With a low target, within 3 minutes after the last invocation with an SMB, the SMB will be blocked, but a temp basal (possibly zero temp) will be set. This might be confusing as the user expects additional insulin (tolerable). This could be remedied by enforcing the 3 minute timewindow using constraints, so the user sees a message (currently _invoke_ only logs this). 
* With a low target, 3-5 minutes after the last invocation a second SMB (for the same BG) might be administered. This would then block an SMB for the next (BG-invoked) loop iteration (since the previous SMB is within the 3 minute window) (tolerable)
* Concurrent invocations are already possibly with the Loop tab, with the regular safeguards preventing harm (ok)

Overall, the benefit would be ~2.5m faster reaction and an immediately visible reaction to the users request.

File: app/src/main/java/info/nightscout/androidaps/plugins/aps/loop/LoopPlugin.java [hunk 1]
```diff
@@ -175,6 +176,14 @@ public long suspendedTo() {
         return loopSuspendedTill;
     }
 
+    @Subscribe
+    public void onStatusEvent(final EventTempTargetChange ev) {
+        new Thread(() -> LoopPlugin.getPlugin().invoke("EventTempTargetChange", true)).start();
```
Comment (jotomo) [inline app/src/main/java/info/nightscout/androidaps/plugins/aps/loop/LoopPlugin.java]: Minor style point: _LoopPlugin.getPlugin()_ is redundant. Calling _invoke_ directly would make the intent easier to read.

**Pull request merged**